weaviate/values.yaml

```diff
@@ -146,21 +146,35 @@ serviceMonitor:
 startupProbe:
   # For kubernetes versions prior to 1.18 startupProbe is not supported thus can be disabled.
   enabled: false
-
+  probeType: httpGet
+  probe:
+    httpGet:
+      path: /v1/.well-known/ready
+      port: 8080
```

Comment:
Can you add also (but commented) an example of command probe?

Reply:
and it would be great to test that you can actually set a different probe, example:
```
check_string_existence "--set startupProbe.probeType: command" "command:"
```